api/data-persistence.ts

```
@@ -0,0 +1,125 @@
+import express, { Request, Response, NextFunction } from 'express';
+import Airtable from 'airtable';
+import { FeatureFlags } from './feature-flags';
+
+const router = express.Router();
+
+interface FeatureFlags {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'FeatureFlags'. Consider to delete it or rename it.

'FeatureFlags' is defined here:


(lint/suspicious/noRedeclare)